tests/autopilot/autopilot_test.go

```diff
@@ -0,0 +1,458 @@
+package tests
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	apapi "github.com/libopenstorage/autopilot-api/pkg/apis/autopilot/v1alpha1"
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/reporters"
+	. "github.com/onsi/gomega"
+	"github.com/portworx/torpedo/drivers/node"
+	"github.com/portworx/torpedo/drivers/scheduler"
+	"github.com/portworx/torpedo/pkg/aututils"
+	. "github.com/portworx/torpedo/tests"
+)
+
+var (
+	testSuiteName            = "AUT"
+	workloadTimeout          = 5 * time.Hour
+	retryInterval            = 30 * time.Second
+	unscheduledResizeTimeout = 10 * time.Minute
+)
+
+var autopilotruleBasicTestCases = []apapi.AutopilotRule{
+	aututils.PvcRuleByUsageCapacity(50, 50, ""),
+	aututils.PvcRuleByUsageCapacity(90, 50, ""),
+	aututils.PvcRuleByUsageCapacity(50, 50, "21474836480"),
```

Comment:
Why is `21474836480` in bytes. Since this is a helper function argument, we should use something like 20Gi here since this is the supported `maxsize` argument format. Will need changes in torpedo where this gets parsed.

We can do that after this PR.